src/types/group.d.ts

```diff
@@ -0,0 +1,21 @@
+type Permission = 'READ' | 'WRITE' | 'DELETE' | 'SHARE' | 'UPLOAD_FILES';
```

Comment:
move the string definitions to the enum or constants as they could also be used in validation